### PR TITLE
Added syncing indicator to tmux status line

### PR DESCRIPTION
**Pull request by erenfro**
This checks for if the panes are synced or not, and provides and
indicator resembling it in  the status bar.

### CODE REVIEW

**Comment by erenfro** [comment]
Woops. Found something I forgot to put in, will update. :)

**Comment by erenfro** [comment]
There we go. I wanted to add in support for synced panes because it's a feature I use in tmux from time to time, but had no visual insight that it was enabled. This adds that visibility notification in, restoring balance to the world once again. :) 

Hope you like it.

**Comment by erenfro** [comment]
Sure. Just for an quick and simple example of look, you can also use this emojipedia link to get an idea of how it would look on various platforms:

https://emojipedia.org/clockwise-rightwards-and-leftwards-open-circle-arrows/

screenshot from 2018-07-20 14-47-25

That's what it looks like on my Linux desktop with noto-emoji fonts.

**Comment by erenfro** [comment]
I'm trying to figure out how to squash this down to a single commit, though. I tried a `git rebase -i HEAD~3` maybe I did something wrong. heh

**Comment by gpakosz** [comment]
Thanks for the emojipedia link!

Yeah you should be able to `rebase -i`, use the "fixup" feature, then `push -f` to your branch and the PR will update itself.

**Comment by erenfro** [comment]
There we go! Hopefully that does a body good, right? ;)

**Comment by gpakosz** [comment]
Looks solid 👌

Do you mind if I wait until Monday to test it on Windows WSL and Cygwin?

PS: thanks for respecting the philosophy of the conf, appreciated 🙌

**Comment by erenfro** [comment]
No problem at all. I've locally committed this to my own mirrored copy, but will wipe it out when things are merged here and my mirror updates. :)
I've been wanting this indicator for a while now, because without it, one can easily forget it's enabled, and do "very bad things" ©️ ™️ accidentally, even of course unintentionally.

**Comment by erenfro** [comment]
Besides that, I forgot the _bg in the actual resulting indicator portion, so I fixed that and committed it in, after testing that functionality more. :)

**Comment by gpakosz** [comment]
image

Yes... It doesn't really look good on Windows ☝️

Maybe the lock symbol `U+1F512` would be a better default? 🔒

image

**Comment by gpakosz** [comment]
Hello @erenfro,

What's your opinion on using `U+1F512` as the default symbol for synchronized panes?

Beside that choice, your PR works great 👍

**Comment by gpakosz** [comment]
Hello @erenfro,

Are you ok if we replace `U+1F501` by `U+1F512` for the default symbol for synchronized panes? `U+1F501` is not displayed correctly in WSL and Cygwin

**Comment by erenfro** [comment]
Yeah, that should be fine. I'll go ahead and do that. It's user customizable anyway. :)

**Comment by erenfro** [comment]
@gpakosz Done. :)

File: .tmux.conf.local [hunk 1]
```diff
@@ -188,6 +189,12 @@ tmux_conf_theme_root_fg='none'
 tmux_conf_theme_root_bg='none'
 tmux_conf_theme_root_attr='bold,blink'
 
+# synchronized indicator
+tmux_conf_theme_synchronized='🔁 '     # U+1F501
```
Comment (gpakosz) [inline .tmux.conf.local]: Woopsy! 🙃

File: .tmux.conf [hunk 2]
```diff
@@ -844,6 +849,9 @@ run 'cut -c3- ~/.tmux.conf | sh -s _apply_configuration'
 #   tmux_conf_theme_status_left=$(echo "$tmux_conf_theme_status_left" | sed \
 #     -e "s%#{root}%#[fg=$tmux_conf_theme_root_fg]#[bg=$tmux_conf_theme_root_bg]#[$tmux_conf_theme_root_attr]#(cut -c3- ~/.tmux.conf | sh -s _root #{pane_tty} #D)#[inherit]%g")
 #
+#   tmux_conf_theme_status_left=$(echo "$tmux_conf_theme_status_left" | sed \
+#     -e "s/#{synchronized}/#[fg=$tmux_conf_theme_synchronized_fg]#[bg=$tmux_conf_theme_synchronized_bg]#[$tmux_conf_theme_synchronized_attr]#{?pane_synchronized,$tmux_conf_theme_synchronized,}/g")
```
Comment (gpakosz) [inline .tmux.conf]: BTW can you use `%` as the line above for the regex delimiter please?

**Comment by erenfro** [comment]
@gpakosz Fixed, and Fixed. :)

**Comment by erenfro** [comment]
Alrighty! Tis done. hehe

**Comment by gpakosz** [comment]
Thank you Eric!

**Pull request merged**